### ISSUE TITLE
Bug with figures

### ISSUE THREAD

**Issue by ejemba** [vandot/casbab#1]
Hi I have a bug when use with figure
Nominal case
```
$ echo epo | ./casbab.sh title
Epo
$ echo $?
0
```
Error case #1
```
$ echo epo2 | ./casbab.sh title
$ echo $?
1
```
Error case #2
```
$ echo 2epo | ./casbab.sh title
$ echo $?
1
```

Can you check this one ?

Thank you

**Comment by vandot** [comment]
It works with figure when there is a delimiter inside string
```
$ echo epo2 epo1 | ./casbab.sh title
Epo2 Epo1
```
```
$ echo 2epo-1epo | ./casbab.sh title
2epo 1epo
```
There is an issue when it tries to find if there is case change to use it as a delimiter.

**Comment by vandot** [comment]
There was a bug for every non alpha char, now it is fixed in **[27bb603](https://github.com/vandot/casbab/commit/27bb60318c623dc1dea0510e76480d74c05a7781)**
